src/context/WatchLaterContext.js

```diff
@@ -0,0 +1,112 @@
+import { useContext, createContext, useState } from "react";
+
+import axios from "axios";
+
+import { useNavigate } from "react-router-dom";
+
+const WatchLaterContext = createContext({});
+
+const WatchLaterProvider = ({ children }) => {
+  const navigate = useNavigate();
+  const [likedVideoList, setLikedVideoList] = useState([]);
+  const [watchLaterList, setWatchLaterList] = useState([]);
+  const [historyList, setHistoryList] = useState([]);
```

Comment:
The file naming should reflect what the file contains.
Either keep these in different files or name the file appropriately

Reply:
Yaa right i have to change the file name will do that towards the end